src/structure.cpp

```diff
@@ -958,6 +959,7 @@ double structure_chunk::max_eps() const {
 }
 
 bool structure::equal_layout(const structure &s) const {
+  std::cout << "equal_layout num_chunks " << num_chunks << " s.num_chunks " << s.num_chunks << "\n";
```

Comment:
Is this something that was left over from debugging and therefore should be removed from this PR? 

In general, for printing to standard output in Meep, use the built-in function `master_printf("equal_layout %d s.num_chunks %d\n",num_chunks,s.num_chunks);` which will only print from the master process in a parallel simulation.

Reply:
Removed this line